extensions/markdown-language-features/src/languageFeatures/copyFiles/copyPasteLinks.ts

```diff
@@ -43,6 +45,14 @@ class PasteLinkEditProvider implements vscode.DocumentPasteEditProvider {
 	}
 }
 
+export function validateLink(urlList: string): boolean {
+	const url = urlList?.split(/\s+/);
+	if (url.length > 1 || !externalUriSchemes.includes(vscode.Uri.parse(url[0]).scheme)) {
```

Comment:
Does `Uri.parse` ever throw? For example if it is passed something that isn't a uri or is a malformed uri?